marklogic-data-hub/src/main/resources/ml-modules/services/delete-jobs.xqy

```diff
@@ -0,0 +1,87 @@
+(:
+  Copyright 2012-2016 MarkLogic Corporation
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+:)
+xquery version "1.0-ml";
+
+module namespace service = "http://marklogic.com/rest-api/resource/delete-jobs";
+
+declare namespace rapi = "http://marklogic.com/rest-api";
+
+declare variable $SUCCESS-KEY := "success";
+declare variable $FAILED-KEY  := "failed";
+
+declare option xdmp:mapping "false";
+
+declare private function service:delete-jobs($job-ids as xs:string*) as map:map
```

Comment:
Please put these private methods in a com.marklogic.hub/job-lib.xqy to be consistent with the other stuff.